lib/web/apiserver.go

```diff
@@ -1423,6 +1423,19 @@ func (h *Handler) siteNodeConnect(
 	log.Debugf("[WEB] new terminal request for ns=%s, server=%s, login=%s, sid=%s",
 		req.Namespace, req.Server, req.Login, req.SessionID)
 
+	authAccessPoint, err := site.CachingAccessPoint()
+	if err != nil {
+		log.Debugf("Unable to auth access point: %v.", err)
+		return nil, trace.Wrap(err)
```

Comment:
use `trace.Wrapf` to add extra text to `err`.
the caller will eventually log this returned `err`, no need to double-log

Reply:
I spoke to @fspmarshall about this issue.

The errors are not being logged by the caller side, which in this case would be the web client. I suppose we could, @alex-kovoy what do you think?

But I think it makes more sense to have server side error logged in the server side.

Reply:
The caller in this case is our middleware (withHanlderXX wrapper) which does not log the errors automatically, so it's up to a handler to do it. 

siteNodeConnect is the handler. 

I agree that it's a bit messy. So lets pass the site (cluster) to the NewTerminal method and retrieve the keep alive values in there. 

Lets do this:
```golang
req.Namespace = namespace
req.ProxyHostPort = h.ProxyHostPort()	
term, err := NewTerminal(req, cluster, ctx)  <-- // do it in here
if err != nil {
     log.Errorf("[WEB] Unable to create terminal: %v", err)
     return nil, trace.Wrap(err)
}
```